examples/advanced/src/example.tsx

```diff
@@ -1125,7 +1127,7 @@ export default class Example extends React.Component<
                 type: MapController,
                 doubleClickZoom: false,
               },
-              legacyMeterSizes: true,
+              // legacyMeterSizes: true,
```

Comment:
Why it's commented?

Reply:
error TS2345 'legacyMeterSizes' does not exist in type 'MapViewProps & CommonViewProps<MapViewState>'